ios/FluentUI/Vnext/PersonaButton/PersonaButton.swift

```diff
@@ -227,29 +228,50 @@ public struct PersonaButton: View {
         Group {
             Text(state.primaryText ?? "")
                 .lineLimit(1)
-                .frame(maxWidth: tokens.labelWidth, alignment: .center)
+                .frame(alignment: .center)
                 .scalableFont(font: tokens.labelFont)
                 .foregroundColor(Color(tokens.labelColor))
             if state.buttonSize.shouldShowSubtitle {
                 Text(state.secondaryText ?? "")
                     .lineLimit(1)
-                    .frame(maxWidth: tokens.labelWidth, alignment: .center)
+                    .frame(alignment: .center)
                     .scalableFont(font: tokens.sublabelFont)
                     .foregroundColor(Color(tokens.sublabelColor))
             }
         }
+        .padding(.horizontal, tokens.padding)
     }
 
     @ViewBuilder
     private var avatarView: some View {
         AvatarView(state.avatarState)
             .padding(.top, tokens.padding)
             .padding(.bottom, tokens.avatarInterspace)
-            .padding(.horizontal, tokens.padding)
+    }
+
+    /// Width of the button is conditional on the current size category
+    private func adjustedWidth() -> CGFloat {
```

Comment:
fluent is usually snaps to 4 px grid. the fact these numbers are not dividable by 4 makes me little concern. also, are we giving too much spacing in certain category? or is it coming from designer recommendation?

Reply:
These numbers come from the CCV, but I don't know where _those_ come from. I'll investigate further before merging the PR.

Reply:
Yes! Because I spent too much of my career working in Objective-C, which doesn't use computed properties the same way :)

Updating now!